static/openapi.yaml

```diff
@@ -276,7 +465,32 @@ paths:
               schema:
                 type: array
                 items: {}
-          description: ''
+          description: 'ExplanationOfBenefit FHIR resources bundle'
+        '400':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+          description: 'Error: Bad Request, e.g. the _lastUpdated operator is not valid.'
+        '401':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+          description: 'Error: Unauthorized, e.g. Authentication credentials were not provided.'
+        '502':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+            application/fhir+json:
+              schema:
+                type: array
+                items: {}
+          description: 'Error: Bad Gateway, e.g. An error occurred contacting the FHIR server'
       tags:
         - v1
   /.well-known/openid-configuration-v2:
```

Comment:
Can openid config return any type of errors?

Reply:
openid config collects info without going to backend, so 502 does not apply here.